lib/rubocop/cop/root_cops/no_tracers.rb

```diff
@@ -0,0 +1,13 @@
+module RootCops
+  class NoTracers < ::RuboCop::Cop::Cop
+    MSG = "Remove all `Tracer` occurrences".freeze
+
+    def_node_matcher :trace_method, <<-PATTERN
+      (send (const _ :Tracer) :trace_method)
```

Comment:
Would it be enough to check for `(const _ :Tracer)` alone?

It occurs to me that `trace_method` might not be the only thing you call on `Tracer`, and we probably don't want any of those other things in the code, either.

Reply:
`(const _ :Tracer)` is not enough. `(send (const _ :Tracer) _)` is. I'll update the matcher and the spec.